packages/mux-player/src/template.ts

```diff
@@ -1,21 +1,14 @@
-import { document } from './polyfills';
 import 'media-chrome/dist/media-theme-element.js';
 // @ts-ignore
 import cssStr from './styles.css';
-// @ts-ignore
-import muxTheme from './media-theme-mux.html';
 import './dialog';
 import { getSrcFromPlaybackId, getStreamTypeFromAttr } from './helpers';
 import { html } from './html';
 import { i18n, stylePropsToString } from './utils';
-
+import classicTheme from './themes/classic';
 import type { MuxTemplateProps } from './types';
 import { StreamTypes } from '@mux/playback-core';
 
-const muxTemplate = document.createElement('template');
-if ('innerHTML' in muxTemplate) muxTemplate.innerHTML = muxTheme;
```

Comment:
question(blocking): Why did you remove this code? I believe it will add slightly faster render time than the refactored version (cc @luwes). If so, I recommend reverting.

Reply:
This has been moved here [`src/themes/classic.ts`](https://github.com/muxinc/elements/pull/668/files#diff-5f2f1a5ac88395921da3dd66c63088cb4e4bcbe6552be1782212e3330f2c8e70R7). It seemed like it belongs to the theme as opposed to the consuming code, is that backwards?

Reply:
I think it's reasonable to keep in classic.ts, since this template element is available on the exported object.

Reply:
this is fine yes, we want to have a template object that can be set via the MediaTheme#template prop